client-js/app/scripts/manager.ts

```diff
@@ -34,13 +34,21 @@ export interface ServerInfo {
   settings: Array<SettingInfo>;
 }
 
+export interface OAuthInfo {
```

Comment:
The names `client_id` and `client_secret` are used throughout the OAuth apis, which is why I like using the full names here (appropriately camel-cased). Can change if you feel strongly.